common/util.go

```diff
@@ -605,3 +605,82 @@ func NewCliHelper() CliHelper {
 		cZones:            make(map[string]string),
 	}
 }
+
+var overrideValueByPrefixLogMapLock sync.Mutex
```

Comment:
I looked at it, but you can't hold the r lock and then take the w lock, so the implementation is a bit ugly (have to check the map twice, once under r lock, second under w lock). I don't expect this to be high contention.